### DISCUSSION TITLE
Merged commit entries not tappable

### DISCUSSION THREAD

**Discussion by anohren** [community/community#3255] [Mobile]
There are some problems with reading a PR in the current version of the iOS app:

- There are references to commit hashes in the timeline, but they're not tappable:

![image](https://user-images.githubusercontent.com/3519879/106180780-0feecc80-619d-11eb-80d0-3c4d345dae61.jpeg)

- Even if you try to remember the hash and go to inspect individual commits in the PR, the details view of a commit doesn't tell you the commit's hash (only its parent's hash). 

Ie the info about which commit has been merged is not useful.

**Comment by jsoref** [comment]
@stevepopovich I was about to file this. Although I'd have summarized it as `Merged commit entries aren't tappable`

**Reply by anohren** [reply]
Probably a better title. I think this was framed as "not even the workaround works" since otherwise someone would inevitably come along and say "You could work around this. You should work around this."

**Reply by jsoref** [reply]
Can you retitle it? :)

**Reply by anohren** [reply]
Sure, done (if you think it'll increase its chances)
